src/app/models/Region.ts

```diff
@@ -0,0 +1,21 @@
+export interface Region {
+  name: string;
+  code: string;
+  latitude: number;
+  longitude: number;
+}
+
+export const regions: Region[] = [
+  {
+    name: 'The world',
+    code: 'global',
+    latitude: 0.0,
+    longitude: 0.0
+  },
+  {
```

Comment:
We could populate this `regions` array with the ones from the countries package. But could do that later.